typescript-operators/main.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-unused-vars */
+
+interface Customer {
+  name: string;
+  details?: {
+    address?: {
+      city: string;
+      street: string;
+      zip: string;
+    };
+    age?: number;
+  };
+}
+
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+const v5: Customer = {
+  name: 'Carl',
+  details: {
+    age: 82,
+  },
+};
+
+console.log('v1 && ', v1 && 'v1 is truthy');
+console.log('v2 && ', v2 && 'v2 is truthy');
```

Comment:
Not quite! This is making it so that your `console.log` is ALWAYS going to run. We only want the console log to run if `v1` or `v2` is truthy, if not, a console log should never happen